packages/syft/src/syft/core/pointer/pointer.py

```diff
@@ -226,9 +227,30 @@ def _get(
             delete_obj=delete_obj,
         )
 
-        obj = self.client.send_immediate_msg_with_reply(
-            msg=obj_msg, timeout=GET_OBJECT_TIMEOUT
+        obj: Any = None
+        is_processing_pointer = self.client.processing_pointers.get(
+            self.id_at_location, False
         )
+
+        while is_processing_pointer:
+            try:
+                obj = self.client.send_immediate_msg_with_reply(
+                    msg=obj_msg, timeout=GET_OBJECT_TIMEOUT, verbose=True
+                )
+                self.client.processing_pointers.pop(self.id_at_location, None)
+                is_processing_pointer = False
+            except UnknownPrivateException:
```

Comment:
This feels like it runs the risk of creating infinite loops which may be difficult to debug at a later date. I feel we should pass in a timeout which defaults to 90 seconds (looking at the tests commented out in this PR) and give an intuitive message if the timeout occurs "Object not found - it may be the that this object doesn't yet exist or it may be that this object has been deleted. If you believe that the object doesn't yet exist, please retry your query momentarily (or set a longer timeout .get(timeout=<insert longer time here>)   ). However, if you worry the object have have been pre-maturely deleted or that an earlier failure may have prevented the object from being created, more advanced interrogation of the system is needed. Start by attempting to re-run your code, checking that each preceeding object exists (you can call my_pointer.exists to check). Hopefully this will show you where an error appears to be occurring. If this does not resolve the issue and you are not an admin, you may need to contact the Domain's administrator for help."